src/main/java/io/hops/metadata/hdfs/entity/FileProvXAttrBufferEntry.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2019 hops.io.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hops.metadata.hdfs.entity;
+
+import com.google.common.base.Charsets;
+
+public final class FileProvXAttrBufferEntry {
+
+  private final long iNodeId;
+  private final byte namespace;
+  private final String name;
+  private final int iNodeLogicalTime;
+  private final String value;
```

Comment:
You should use byte[] for value and then provide the method to convert to string when needed. Check the StoredXAttr, to be consistent with it

Reply:
ok, switched to a PrimaryKey and byte[] value, mirroring StoredXAttr.